<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApInfomation</class>
 <widget class="QFrame" name="ApInfomation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>SSID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lab_SSID">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>text</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>加密方式</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lab_safty">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>频率</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lab_freq">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>信号强度</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="lab_signallevel">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>text</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>连接速度</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="lab_bitrate">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>连接状态</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="lab_connect_status">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>未连接</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>本机mac地址</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lab_pd_mac">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>text</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>IP地址</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="lab_IP">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>text</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>子网掩码</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="lab_mask">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>text</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>密码保存</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="lab_saveconnect">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>未保存</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="lab_gate">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>text</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_12">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>路由器</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>mac地址</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lab_mac">
         <property name="styleSheet">
          <string notr="true">color:white</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_forget">
       <property name="text">
        <string>忘记网络</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbt_close">
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
